cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://staging-ddw.devinit.org",
+  "baseUrl": "http://localhost/",
```

Comment:
You need to be careful what you commit ... at the very least, review the files you stage before you commit them.